### ISSUE TITLE
"Back to top" defects

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#24752]
# Issue Description
This issue was found during testing in preparation for staging review. 

**Defect**
"Back to top" component has several defects.
 - All page content should be contained by landmarks
 - ARIA hidden defect

![aira hidden back to top defect](https://user-images.githubusercontent.com/55411834/118336755-8ce54e00-b4cf-11eb-942e-bee773145557.png)

![image](https://user-images.githubusercontent.com/55411834/118336863-c3bb6400-b4cf-11eb-88d1-dc7b0564c8b7.png)

**Code**
```
<div id="back-to-top-container" class="">
<div class="usa-content">
<button type="button" classname="usa-button" class="va-top-button-transition-in">
<span>
<i aria-hidden="true" class="fas fa-arrow-up" role="img"></i>
</span>
<span>
Back to top
</span>
```

---
## Tasks
- [ ] 
## Acceptance Criteria
- [ ] Veterans using screenreaders know they are able to use the "back to top" component to return to the top of the page.

**Comment by mmiddaugh** [comment]
Accessibility feedback noted in January in #18399
